packages/migrations/src/types.ts

```diff
@@ -27,3 +28,38 @@ export enum ContractName {
     EtherDelta = 'EtherDelta',
     Arbitrage = 'Arbitrage',
 }
+
+export interface LedgerCommunicationClient {
+    close: () => Promise<void>;
+}
+
+export interface LedgerGetAddressResult {
+    address: string;
+    publicKey: string;
+    chainCode: string;
+}
+
+export interface ECSignatureString {
+    v: string;
+    r: string;
+    s: string;
+}
+
+export interface LedgerGetAddressResult {
+    address: string;
+    publicKey: string;
+    chainCode: string;
+}
+
+export interface LedgerEthereumClient {
```

Comment:
This is not exported, but that's an oversight. Export it from `@0xproject/subproviders` and import it from there.